gustaf/utils/comm.py

```diff
@@ -0,0 +1,70 @@
+"""
+Experimental communication utilities.
+WebSocketClient requires `websockets` packages and python 3.7+.
```

Comment:
Since Python < 3.8 is in `end of life` is this necessary? If you want to keep support for Python 3.6 you should use the ImportError handling we have in gustaf.

Reply:
you mean l.13 ?

Reply:
Good point and I think that might be something for other PR